recipe/0001-proj-6.3.0-tests.patch

```diff
@@ -0,0 +1,270 @@
+From 82f01bdcd0cabd4a43b956db66c4b50d2c3351fb Mon Sep 17 00:00:00 2001
+From: "Alan D. Snow" <alansnow21@gmail.com>
+Date: Wed, 1 Jan 2020 20:53:57 -0600
+Subject: [PATCH] update CICD builds with 6.3.0 (#500)
```

Comment:
I took out the patches related to their CI, so this info isn't strictly accurate.  I'm assuming no one is concerned because this patch is likely to be short-lived.